include/swift/AST/SILOptions.h

```diff
@@ -121,7 +122,9 @@ class SILOptions {
   bool AssumeUnqualifiedOwnershipWhenParsing = false;
 
   /// Assume that code will be executed in a single-threaded environment.
-  bool AssumeSingleThreaded = false;
+  /// TODO: return to false when we pass "-Xfrontend -assume-single-threaded"
+  ///       to swift compiler while building standard library.
+  bool AssumeSingleThreaded = std::getenv("SWIFT_ASSUME_SINGLE_THREADED");
```

Comment:
I have already added -swift-stdlib-use-nonatomic-rc option to build-script-impl. I will remove std::getenv once I confirm that the option has the same effect with it.